src/libopensc/pkcs15-iasecc.c

```diff
@@ -103,34 +103,32 @@ _iasecc_md_update_keyinfo(struct sc_pkcs15_card *p15card, struct sc_pkcs15_objec
 	LOG_FUNC_RETURN(ctx, rv);
 }
 
-
 /*
  * CPx cards have an undocumented issue: they lack of
  * Algorithm's reference into their PKCS's ASN1 encoding.
  */
 static int
 _iasecc_cpx_fixup_prkdf(struct sc_pkcs15_card *p15card)
 {
-	struct sc_context * const ctx = p15card->card->ctx;
+	struct sc_context *const ctx = p15card->card->ctx;
```

Comment:
shouldn't this be `const struct sc_context *ctx`?

Reply:
right. Missed that when rewriting too many changes. I will fix it.